File: Program/python_and_ruby_101/Function/5.py
input_id = input("id를 입력해주세요 \n")

def login(_id):
    members = ['egoing','k8805']
    for member in members :
        if(member == _id) :
            return True
    return False

if login(input_id) :
    print('hello '+input_id)
else :
    print('who are you?')

File: Program/python_and_ruby_101/Function/5.rb
puts("id를 입력해주세요")
input_id = gets.chomp()

# members =['egoing','k8805','leezche']
#
# for member in members do
#   if member == input_id
#     puts("Hello! " + input_id)
#     exit
#   end
# end
# puts("Who are you?")

def login(_id)
  members =['egoing','k8805','leezche']
  for member in members do
    if member == _id
      return true
    end
  end
  return false
end

if login(input_id)
  puts("hello "+input_id)
else
  puts("Who are you?")
end
